jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java

```diff
@@ -128,17 +128,10 @@ public void initColumns(ExtendedPagedTable extendedPagedTable) {
                                          constants.Version()));
         columnMetas.add(new ColumnMeta<>(projectColumn,
                                          constants.Project()));
-        final ColumnMeta columnMeta = new ColumnMeta<>(actionsColumn,
-                                                       "");
-        final TextHeader header = new TextHeader(constants.Actions());
-        header.setHeaderStyleNames("text-center");
-        columnMeta.setHeader(header);
-        columnMetas.add(columnMeta);
+        columnMetas.add(actionsColumnMeta);
 
+        extendedPagedTable.addSelectionIgnoreColumn(actionsColumnMeta.getColumn());
         extendedPagedTable.addColumns(columnMetas);
-        extendedPagedTable.setColumnWidth(actionsColumn,
```

Comment:
I believe we still need to set the column width for actions, to match the other list tables.